src/gz/z64.h

```diff
@@ -389,7 +389,9 @@ typedef struct
   uint16_t      timer;              /* 0x13D2 */
   char          unk_11_[0x0035];    /* 0x13D4 */
   uint8_t       language;           /* 0x1409 */
-  char          unk_12_[0x0046];    /* 0x140A */
+  char          unk_12_[0x0002];    /* 0x140A */
+  uint8_t       target;             /* 0x140C */
```

Comment:
the name `target` doesn't not make it clear what this variable does; i suggest `z_targeting` or `target_mode`